core/templates/domain/objects/UnitsObjectFactory.ts

```diff
@@ -130,13 +134,13 @@ export class UnitsObjectFactory {
   convertUnitDictToList(unitDict: UnitsDict): Unit[] {
     var unitList: Unit[] = [];
     for (var key in unitDict) {
-      unitList.push({unit: key, exponent: unitDict[key]});
+      unitList.push({ unit: key, exponent: unitDict[key] });
     }
     return unitList;
   }
 
   unitToList(unitsWithMultiplier: [string, number][]): Unit[] {
-    var unitDict = {};
+    var unitDict: { [key: string]: number } = {};
```

Comment:
Probably separate this into a type variable.

Reply:
done